src/bentoml/_internal/service/service.py

```diff
@@ -435,6 +447,15 @@ def add_grpc_interceptor(
     def add_grpc_handlers(self, handlers: list[grpc.GenericRpcHandler]) -> None:
         self.grpc_handlers.extend(handlers)
 
+    def init_local(self):
+        """
+        Initiate this service instance in local mode, which will load all runners in
+        current Python process. This should be used for one-off inference task, testing
+        and debugging purpose only.
+        """
+        for runner in self.runners:
+            runner.init_local(quiet=True)
```

Comment:
I don't think this is a good idea. Maybe we can make a runners_map and then allow users to do something like

```python
svc = bentoml.Service(runners=[runners_1, runners_2])

svc.runner_1.init_local()
svc.runner_2.init_local()
```

Reply:
users can already do `svc.runners[0].init_local()`, this is just for making it easier to test things in a notebook/dev environment

Reply:
right, but I feel like users shouldn't be exposed to these types of API,  since they will "shoot" themselves in the foot.